client/my-sites/site-settings/site-icon-setting/index.jsx

```diff
@@ -104,9 +149,16 @@ class SiteIconSetting extends Component {
 									isVisible />
 							) }
 							siteId={ siteId }
-							onClose={ this.setSiteIcon }
+							onClose={ this.editSelectedMedia }
 							enabledFilters={ [ 'images' ] }
+							imageEditorProps={ {
+								allowedAspectRatios: [ AspectRatios.ASPECT_1X1 ],
+								onDone: this.setSiteIcon
+							} }
 							visible
+							labels={ {
+								confirm: translate( 'Continue' )
```

Comment:
Should we add a context ? or are there any guidelines to know when we need to add a context or not